src/test/resources/application.properties

```diff
@@ -12,4 +12,17 @@ cors.allowedHeaders=Authorization,Cache-Control,Content-Type
 jwt.secret.key=QdzigVY4XWNItestqpRdNuCGXx+FXok5e++GeMm1OlE=
 jwt.token.expiration=#{1000 * 60 * 24}
 
-spring.flyway.enabled=false
\ No newline at end of file
+spring.flyway.enabled=false
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
+
+## File Storage Properties
+# All files uploaded through the API will be stored in this directory
+image.folder.path=/tmp/Jms/Images/
+
+# Enable multipart uploads
+spring.servlet.multipart.enabled=true
+# Max file size
+spring.servlet.multipart.max-file-size=12MB
+# Max Request Size
+spring.servlet.multipart.max-request-size=12MB
+multipart.file.max.size=8
```

Comment:
If these are the same as the ones in the src/main/resources/application.properties file, then we shouldn't need to specify them here as well